packages/test-project/src/components/OldApiExample.vue

```diff
@@ -15,31 +15,31 @@
     <p>X + Y = {{ xPlusY }}</p>
     <div style="background: rgba(219, 53, 53, 0.62); color: #ff9090; padding: 10px 15px">
       <p
-        v-for="(error, index) of $v.$errors"
+        v-for="(error, index) of vv.$errors"
```

Comment:
So are we keeping it v or vv? Vv is like veevalidate

Reply:
Oh dunno. Just needed something to skip the `$` prefix.